tests/integration/conftest.py

```diff
@@ -78,11 +79,11 @@ async def _factory(contract_app, contract_name, *args, **kwargs):
     return _factory
 
 
-Wallet = namedtuple("Wallet", ["address", "private_key", "starknet_address"])
+Wallet = namedtuple("Wallet", ["address", "private_key", "starknet_contract"])
```

Comment:
looking at the test, the sequence `owner.starknet_contract.contract_address` seems a bit too long and reduce readability (it's already quite long and disturbing with `caller_address=owner.starknet_address` IMHO).

So I'd suggest that either we add the `starknet_address` again:
```
Wallet = namedtuple("Wallet", ["address", "private_key", "starknet_contract", "starknet_address"])
```

or we update `wrap_for_kakarot` to have a `contract.connect(owner: Wallet)` method and remove all these extra char in all the tests